ui/admin/app/styles/app.scss

```diff
@@ -698,3 +698,35 @@
   min-height: 1rem;
   margin: 1.125rem 0 1.5rem;
 }
+
+.mapping-table {
+  /* stylelint-disable selector-class-pattern */
+  td > div,
```

Comment:
Non blocking comment: I am fine disabling this rule, but asking for curiosity, it has the `td > div, .hds-form-field__control:not(:first-child)` selector that cause the linter to complain?

Reply:
It's the usage of `__` by HDS classes. I had to use some of the HDS classes as otherwise I couldn't quite get the right CSS specificity to override their styles correctly.

Open to other suggestions.

Reply:
Ahh I see, I was wondering what was the cause, now we know is HDS classes and eventually we might need to do this again, just to be aware team wise 😉

Reply:
Can we use class names rather than anonymous html elements for clarity?

Reply:
if these table style changes are not just specific to credential library resource and applicable to boundary UI overall, we could move these to `override.scss` where we override HDS styles. We did a similar thing for fieldsets recently.

Reply:
Hmm they're not for boundary overall, just the usage of their table in this instance, I'll try using class names though; I think I had some difficulty with CSS specificity but that wasn't with my own classes.

Reply:
I wasn't able to override correctly the HDS styles using my own classes still (without using !important), not sure if you have suggestions for this.